config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_spannerinstances.spanner.cnrm.cloud.google.com.yaml

```diff
@@ -77,10 +77,10 @@ spec:
                   in length.
                 type: string
               numNodes:
-                format: int64
+                format: int32
                 type: integer
               processingUnits:
-                format: int64
+                format: int32
```

Comment:
I don't remember the details, could you verify against real k8s cluster to see if we can change this? 
e.g. 
1. `kubectl apply crds/<old-spanner-int64>` 
2. `kubectl apply -f <example.yaml using processingUnits int64>` 
3. `kubectl apply crds/<new-spanner-int32>`

Reply:
That's awesome, thank you!